src/components/charts/shared/observationsTables/ObservationsTable.jsx

```diff
@@ -71,9 +82,9 @@ class ObservationsTable extends React.PureComponent {
         )}
         <Table
           className="observations-table"
-          colTitles={colTitles}
+          colTitles={this.translateCells(colTitles)}
```

Comment:
translation has been moved entirely out of low level components like `Table` and is instead performed in it's wrappers like `ObservationsTable` to avoid duplicate translation and mixing translation code with unrelated code.